api-analytics/index.js

```diff
@@ -13,12 +11,15 @@ document.getElementById('add-form').onsubmit = function() {
   if (!newQuote) { return; }
 
   // Leave blank to represent an anonymous quote.
-  var quoteAuthor = document.getElementById('quote-author').value.trim() || 'Anonymous';
+  var quoteAuthor = document.getElementById('quote-author').value.trim()
+                    || 'Anonymous';
   var quote = { text: newQuote, author: quoteAuthor };
   var headers = { 'content-type': 'application/json' };
 
   // Send the API request. In this case, a `POST` on `quotations` collection.
-  fetch(ENDPOINT, { method: 'POST', body: JSON.stringify(quote), headers: headers })
+  fetch(ENDPOINT, {
+    method: 'POST', body: JSON.stringify(quote), headers: headers
```

Comment:
Nit: Since you need to wrap this, could you make it:
```
fetch(ENDPOINT, {
  method: 'POST',
  body: JSON.stringify(quote),
  headers: headers,
```